ios_app_clip/README.md

```diff
@@ -0,0 +1,17 @@
+# ios_app_clip
```

Comment:
Can you flesh this out just a bit using the format from the other READMEs? Nothing huge, just the same headers and footer, really:

https://github.com/flutter/samples/tree/master/jsonexample

Reply:
ok